web/src/components/newsList.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import loadable from '@loadable/component';
+import { slidesToShowPlugin, arrowsPlugin } from '@brainhubeu/react-carousel';
+
+import NewsCard from './newsCard';
+import MoreButton from './moreButton';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+export default function NewsList({ news }) {
+    const text = 'More ❯';
+    const back = '❮ Back';
+
+    return (
+        <div className="bg-brand-1 h-full mb-48 pb-48">
+            <h1 className="h1-heading-accent pt-12 text-center text-white text-4xl uppercase">
```

Comment:
This should not be an `<h1>`. There should only ever be 1 `<h1>` per page. If this is the heading for this section then `<h2>` is semantically correct.